### DISCUSSION TITLE
issue with going to next screen

### DISCUSSION THREAD

**Discussion by stan928** [ankitpokhrel/jira-cli#690] [Q&A]
I have a issues list that returns ~500 elements...  when I run jira issues list I only get the first 100, how to I get to the next page.... I feel I must be missing something.

**Comment by ankitpokhrel** [comment]
Hi @stan928, this is not directly possible from the UI yet. I do plan to revamp the TUI hopefully this year which will allow you to iterate between pages + easily filter items from the TUI.
